sys/include/net/gnrc/mac/internal.h

```diff
@@ -0,0 +1,86 @@
+/*
+ * Copyright (C) 2015 Daniel Krebs
```

Comment:
@miri64 , you mean add INRIA into copyright? (I have already done, [here](https://github.com/RIOT-OS/RIOT/pull/6074/files#diff-f45d07c6ad9052e33403063abd1028acR3) )